src/simulation/stage_update.jl

```diff
@@ -31,7 +31,7 @@ end
 
 # This makes the choice in which variable to get from the results.
 function _get_stage_variable(chron::Type{RecedingHorizon},
```

Comment:
not part of this review, but you aren't using `chron`.  Could just be `::Type{Consecutive}`.  And below, too.

Reply:
Not updated yet.